### DISCUSSION TITLE
rewriting url domain of nc aio instances.

### DISCUSSION THREAD

**Discussion by Halokk424** [nextcloud/all-in-one#5140] [Questions] [answered]
Hello, i  have an issue of rewriting the aio url from:

https://domain.com to https://www.domain.com

how can i manage to  achieve this in the aio instances, any ideas?

thank you

**Comment by szaimen** [comment, marked as answer]
Hi, maybe you are looking for this? https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-change-the-domain
